server.js

```diff
@@ -1,33 +1,51 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+import express from "express"
+import bodyParser from "body-parser"
+import cors from "cors"
+import booksData from "./data/books.json"
+
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
-app.use(cors())
+app.use(cors());
 app.use(bodyParser.json())
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// Routes
+app.get("/", (req, res) => {
+  res.send("Books API")
+});
+
+app.get("/books", (req, res) => {
+  res.json(booksData)
+});
+
+app.get("/books/:id", (req, res) => {
+  const bookId = req.params.id
+  const book = booksData.find(item => item.bookID.toString() === bookId)
+  res.json(book)
+})
+
+app.get("/language/:language", (req, res) => {
+  const language = req.params.language
+  const booksByLanguage = booksData.filter(item => item.language_code === language)
+  res.json(booksByLanguage)
+})
```

Comment:
add this as a query to the /books endpoint instead. To either get all books, or if you add a query such as `books?language=swedish` you filter out them and return the filtered books instead.

Same with title further down.